### DISCUSSION TITLE
Need help changing color of the status bar

### DISCUSSION THREAD

**Discussion by sandorex** [EdenEast/nightfox.nvim#335] [Q&A] [answered]
Hello i love the carbonfox theme but i cannot figure out how to change the status bar color, would appreciate help

**Comment by EdenEast** [comment]
What are you using for your statusline?

If you are using lualine as an example then it should work automatically. In the readme there is a [note](https://github.com/EdenEast/nightfox.nvim#lualine) on lualine. It is required to set the colorscheme before calling `lualine.setup`.

This is because lualine checks the current theme (stored in the `vim.g.colors_name` variable) and then tries to load a theme with `require("lualine.themes." .. vim.g.colors_name)`. Nightfox defines this [module](https://github.com/EdenEast/nightfox.nvim/blob/main/lua/lualine/themes/carbonfox.lua).

**Reply by sandorex** [reply]
I was using bare neovim statusline

**Reply by EdenEast** [reply, marked as answer]
Ah if you are using the bare neoivm statusline then there are two highlight groups that you should look at `:h hl-StatusLine` and `:h hl-StatusLineNC`

Here is a quick snippet of setting the background color to blue as an example:

```lua
require("nightfox").setup({
  groups = {
    all = {
      StatusLine = { fg = "palette.bg1", bg = "palette.blue.base" },
    },
  },
})
```
